### PR TITLE
Move quota and enable by Kustomize

### PR DESCRIPTION
**Pull request by vpavlin**
A follow up PR for #321 to improve handling of the quota resource and fix operator doc

@adrian555 Sorry that it did not make it into previous PR:)

### CODE REVIEW

File: operator.md [hunk 1]
```diff
@@ -21,7 +22,6 @@ we use ResourceQuota to provide constraints that only one instance of kfdef is a
 ```shell
 KUBEFLOW_NAMESPACE=kubeflow
 kubectl create ns ${KUBEFLOW_NAMESPACE}
-# kubectl create -f deploy/crds/kfdef_quota.yaml -n ${KUBEFLOW_NAMESPACE} # only deploy this if the k8s cluster is 1.15+ and has resource quota support
```
Comment (animeshsingh) [inline operator.md:24]: Any reason for removing this?

File: kustomization.yaml [hunk 2]
```diff
@@ -1,2 +1,8 @@
-bases:
-- kustomize/base
\ No newline at end of file
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- kustomize/base
+# only enable this if the k8s cluster is 1.15+ and has resource quota support
```
Comment (animeshsingh) [inline kustomization.yaml:5]: neverming - see you have added here. Though should it get reflected in the doc somewhere?

Comment (vpavlin) [reply kustomization.yaml:5]: It is in the doc, I just moved it to the operator deployment along with other kustomize commands

**Comment by vpavlin** [comment]
@adrian555 PTAL

**Comment by adrian555** [comment]
Thanks @vpavlin looks good.

/lgtm

**Pull request merged**